src/bootstrap/src/core/build_steps/compile.rs

```diff
@@ -550,6 +550,10 @@ pub fn std_cargo(builder: &Builder<'_>, target: TargetSelection, stage: u32, car
         cargo.rustflag("-Cforce-unwind-tables=yes");
     }
 
+    // Enable frame pointers by default for the library. Note that they are still controlled by a
+    // separate setting for the compiler.
+    cargo.rustflag("-Cforce-frame-pointers=yes");
+
```

Comment:
I wonder if we would ever need to disable this. What about adding a new option (something like `std-frame-pointers`) to build config that manages this (which will be `true` by default)? Having the ability to easily disable it without having to manipulate the codebase would be valuable for devs.

Reply:
I think it would be useful to get anyone who wants to disable it to open an issue and say why, that would be useful feedback. I'm not sure whether it's worth adding options for "maybe someone needs it".

Reply:
Right, makes sense.